### DISCUSSION TITLE
What about these bundle-xxxxxxxxxx-min.css/js files?

### DISCUSSION THREAD

**Discussion by IngoBelka** [datenstrom/community#547] [Ask a question] [answered]
Hej,

I'm going to build a website with Yellow - thank you, that you give it to the world. Till now it goes very well.

Now I find in the extensions path a lot of these files: bundle-xxxxxxxxxx-min.css/js. Do I need them all? Or is it neccessary to clean up sometimes and how should I do that?

regards Ingo

**Comment by schulle4u** [comment, marked as answer]
Yes, you can safely clean up these files if things become messy. The bundle extension will recreate them next time you open your site in the browser. If your web host offers shell access, a simple one-liner like `rm bundle-*` in your extensions folder should work.

**Comment by IngoBelka** [comment]
@schulle4u 
Thank you for your answer. So I'll go to clean up.

**Comment by schulle4u** [comment]
And Yellow even offers a builtin [commandline option](https://github.com/datenstrom/yellow-extensions/tree/master/source/command) to clean up static files. `php yellow.php clean` also wipes away the bundle files.

**Comment by IngoBelka** [comment]
Oh, well, that's good to know, Thank you very much!

I do not have shell access to the webspace, but I'm doing the developement on a locally running machine. There I can use the command line.
